index.html

```diff
@@ -1,20 +1,61 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>AJAX Project</title>
-  <link rel="icon" href="images/favicon.ico">
-  <link rel="stylesheet" href="css/reset.css">
-  <link rel="stylesheet" href="css/layout.css">
-  <link rel="stylesheet" href="css/styles.css">
-</head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
-  <script src="js/data.js"></script>
-  <script src="js/main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Wester US National Parks Project</title>
+    <link rel="icon" href="images/favicon.ico" />
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/layout.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Chakra+Petch:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&family=Julius+Sans+One&family=Press+Start+2P&family=Sedan+SC&display=swap"
+      rel="stylesheet" />
+  </head>
+  <body>
+    <main>
+      <form>
+        <div class="container">
+          <div class="overlay">
+            <div class="row">
+              <h1>Your Destination</h1>
+              <hr />
+            </div>
+            <div class="row">
+              <label for="states"
+                >Select a State:<span>(within Western US)</span></label
+              >
+            </div>
+            <div>
+              <select id="states" name="western states">
+                <option value="defaultVal"></option>
```

Comment:
You should add both the `selected` and `disabled` attributes to make this selected by default but not selectable